src/test/configs/dockerfile-with-parallel-commands/Dockerfile

```
@@ -0,0 +1,13 @@
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+#  Licensed under the MIT License. See License.txt in the project root for license information.
+ARG VARIANT="16-bullseye"
+
+# Target should skip this layer
+FROM alpine as false-start 
+
+FROM mcr.microsoft.com/vscode/devcontainers/typescript-node:0-${VARIANT} as desired-image
```

```suggestion
FROM mcr.microsoft.com/devcontainers/typescript-node:0-${VARIANT} as desired-image
```
Can we use the new (migrated) image path here?
Reference - https://github.com/devcontainers/images/tree/main/src/typescript-node#using-this-image